README.md

```diff
@@ -1,10 +1,44 @@
 # Project Template
 
-This repository contains code for our groups CSC207 project, a Task and Event Management application.
+This repository contains code for our groups CSC207 project, a Task and Event Management application. Through our program, a user can
+manage their weekly
+1. __Tasks__ which can be thought of as "to-do"s that don't have any timeframe attached to them. For example, 'Bake Cookies' is a task.
+2. __Events__ which have a specified start and end time, unlike tasks. For example, 'CSC207 Lecture from 1 to 2 PM' is an event.
 
-## Checklist For Your Project
+Our program allows the user to Create Tasks and Events, Modify Tasks and Events, Mark Tasks as complete or incomplete, Delete Tasks and Events and clear their weekly schedule in case the user wants to start from scratch.
 
+# Instructions
 
-### IMPORTANT
+Clone the ```main``` branch of the code base. Invoke ```Main.java``` in the ```src/main/java``` directory. We have provided the following GIFs, which demonstrate each use case, for the user's convenience.
 
-Please note that the test case ```testModifyTask()``` in the file ```ModifyTaskControllerTest.java``` is not identified by GitHub workflows and hence, never executed. T. The test case executes (and passes) on the computer of the author, POPPININ (Raghav Arora). However, ```testModifyTask()``` is also not recognized as a unit test (i.e. IntelliJ doesn't show the ```testModifyTask()``` option when one right clicks on the test) on the devices of group members other than the test case author. This problem was discussed with a TA on November 21, 2022.
\ No newline at end of file
+### Create Event
+
+![Create Event](images/CreateEvent.gif)
+
+### Create Task
+
+![Create Event](images/CreateTask.gif)
+
+### Modify Event
+
+![Create Event](images/ModifyEvent.gif)
+
+### Modify Task
+
+![Create Event](images/ModifyTask.gif)
+
+### Delete Event
+
+![Create Event](images/CreateEvent.gif)
+
```

Comment:
Delete event is currently showing the create event gif

Reply:
Good catch Ricky

Reply:
Thank you for catching that Ricky. Fixed it.